.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # これがないとリリースを作成できない
+    steps:
+      # チェックアウト
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
ですかね

```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, actions/setup-go@v3, golangci/golangci-lint-action@v2. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```
がwarningが出るので